### PR TITLE
Chore/DF-20673 superstate time

### PR DESCRIPTION
**Pull request by mmcallister-cll**
## Description
Updating time from 9:09-10:30AM ET to 9:08 AM - 12:00 PM ET

## Steps to Test
1. sanity check request to endpoint succeeds
2. yarn test superstate

## Quality Assurance

- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `adapter-secrets` configuration file or update the [soak testing blacklist](/packages/scripts/src/get-changed-adapters/soakTestBlacklist.ts).
- [ ] If a new adapter was made, or a new endpoint was added, update the `test-payload.json` file with relevant requests.
- [ ] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Jira.
- [ ] This is related to a maximum of one Jira story or GitHub issue.
- [ ] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types).
- [ ] All code changes have 100% unit and integration test coverage. If testing is not applicable or too difficult to justify doing, the reasoning should be documented explicitly in the PR.

[DF-20673]: https://smartcontract-it.atlassian.net/browse/DF-20673?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .changeset/brown-ducks-beg.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/superstate-adapter': patch
```
Comment (mxiao-cll) [inline .changeset/brown-ducks-beg.md:2]: minor

File: packages/sources/superstate/src/transport/nav.ts [hunk 2]
```diff
@@ -78,11 +80,11 @@ export class NavTransport implements Transport<BaseEndpointTypes> {
     return this.execute(fundId, reportValue)
   }
 
-  // Runs 'execute' function every day at 9:09 AM ET (if fundIdsSet is not empty)
+  // Runs 'execute' function every day at START_TIME: 9:08 AM ET (if fundIdsSet is not empty)
   runScheduler() {
     const rule = new schedule.RecurrenceRule()
     rule.hour = 9
-    rule.minute = 9
+    rule.minute = 8
```
Comment (mxiao-cll) [inline packages/sources/superstate/src/transport/nav.ts:87]: Can we parse these from START_TIME

**Review by mxiao-cll — APPROVED** [review]

File: .changeset/brown-ducks-beg.md (hunk 1 above)
Comment (mmcallister-cll) [reply .changeset/brown-ducks-beg.md:2]: you think? It'll still work

File: packages/sources/superstate/src/transport/nav.ts (hunk 2 above)
Comment (karen-stepanyan) [reply packages/sources/superstate/src/transport/nav.ts:87]: It would be ideal if we can have those as env vars as well.

Comment (mmcallister-cll) [reply packages/sources/superstate/src/transport/nav.ts:87]: probably, yeah 😢

File: .changeset/brown-ducks-beg.md (hunk 1 above)
Comment (mxiao-cll) [reply .changeset/brown-ducks-beg.md:2]: minor means you can't go back once upgrade, patch means you can

Comment (mxiao-cll) [reply .changeset/brown-ducks-beg.md:2]: (not important)

Comment (mmcallister-cll) [reply .changeset/brown-ducks-beg.md:2]: i don't feel strongly either way 🤷. I guess it does affect the functionality so i'll use minor

**Review by mxiao-cll — APPROVED** [review]

**Pull request merged**